src/apps/schedule/crud/schedule_history.py

```diff
@@ -15,6 +15,11 @@
 
 
 class ScheduleHistoryCRUD(BaseCRUD[EventHistorySchema]):
+    schema_class = EventHistorySchema
+
+    async def get_by_id(self, id: uuid.UUID) -> EventHistorySchema | None:
+        return await self._get("id", id)
```

Comment:
Sorry I wasn't clearer about this in the ticket description, but the primary key in this table is `id_version`
```suggestion
    async def get_by_id(self, id_version: str) -> EventHistorySchema | None:
        return await self._get("id_version", id)
```